tests/Common/SpanIntegrationChecker.php

```diff
@@ -16,6 +17,10 @@ final class SpanIntegrationChecker
     public function defineIntegrationsByPattern()
     {
         // <regex pattern> => <integration class>
+        $pdoIntegration = 'DDTrace\Integrations\PDO\PDOIntegration';
```

Comment:
Consider ternary instead?

Reply:
Ternary will make us check `Configuration::get()->isSandboxEnabled()` for every integration we add sanboxing for. I chose this way so that we can just group all the sandboxed integrations into one `if` block. :)